lib/cacheable/response_cache_handler.rb

```diff
@@ -68,25 +67,25 @@ def cacheable_info_dump
     end
 
     def try_to_serve_from_cache
-
       # Etag
       serve_from_browser_cache(versioned_key_hash)
+      return if defined?(@response)
```

Comment:
Would it make sense to return the response from these methods or nil if they aren’t applicable, and then pass it through as the return value here as well? Or perhaps return some truthy value iff the response was set and write `and return`

Reply:
I tried some of these options and imo none of them look much better than this